Update migrate-monolith-content.md (#4463)
MicrosoftDocs/architecture-center: docs/microservices/migrate-monolith-content.md

To help facilitate this decomposition, a viable software development approach is to apply the principles of domain-driven design (DDD).

> Domain Driven Design (DDD) is a software development approach first introduced by [Eric Evans](http://domainlanguage.com/ddd/). DDD opposes the idea of having a single unified model for the entire system; instead it encourages dividing the system into bounded contexts, each of which have their own model. DDD requires a good understanding of the domain for which the application will be written. The necessary domain knowledge to create the application resides within the people who understand it &mdash; the domain experts.

Reviewer: ```suggestion
Domain Driven Design (DDD) is a software development approach first introduced by [Eric Evans](http://domainlanguage.com/ddd/). DDD opposes the idea of having a single unified model for the entire system. Instead, it encourages dividing the system into bounded contexts, each of which has its own model. DDD requires a good understanding of the domain for which the application will be written. The necessary domain knowledge to create the application resides within the people who understand it &mdash; the domain experts.
```
